src/Halogen/Store/Select.purs

```diff
@@ -29,3 +33,20 @@ selectEq = Selector <<< { eq, select: _ }
 -- | Create a `Selector` for the entire store.
 selectAll :: forall store. Selector store store
 selectAll = Selector { eq: unsafeRefEq, select: identity }
+
+-- | Maps an `Emitter` using a `Selector`. The resulting `Emitter` will behave
+-- | as if it were `map`ped with the `Selector`'s `select` function, with the
+-- | exception that it will not fire multiple times in a row if the selected
+-- | value has not changed (as determined by the `Selector`'s `eq` function).
```

Comment:
This reads a little more clearly to me. What do you think?

```suggestion
-- | Apply a `Selector` to an `Emitter` so that the emitter only fires when the
-- | selected values change, as determined by the selector's equality function.```

Reply:
```suggestion
-- | Apply a `Selector` to an `Emitter` so that the emitter only fires when the
-- | selected value changes, as determined by the selector's equality function.
```